Sources/Amplitude/Storages/PersistentStorage.swift

```diff
@@ -77,6 +80,13 @@ class PersistentStorage: Storage {
         var content: String?
         do {
             content = try String(contentsOf: eventBlock, encoding: .utf8)
+            if content?.hasSuffix(PersistentStorage.DELMITER) == true {
```

Comment:
Wanted to add this comment in the android one too but forgot. Can we just name the file as -v2 so we know its a v2 file instead of looking for some suffix in the file? What if writing the suffix failed? We'll parse the entire file as v1?

Reply:
I guess we could rename the file as -v2. Right now if writing the suffix failed, we will parse as v1. Hard to call the -v2 is suffix though as we could have split or rename conflict resolve which are adding suffix. Prefix might be safer if we would like to change the name

Reply:
yea prefix works too. As long as we look at the file name instead of some special character in the contents.

Reply:
Yeah, updated this to use prefix